contact.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Products</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<header>
+    <nav>
+        <div class="navigation">
+       <img class= "logo" src="img/lambda-black.png" alt="">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="products.html">Products</a>
+        <a href="">Blog</a>
+        <a href="contact.html">Contact</a>
+    </div>
+    </nav>
+</header>
+
+<body>
+        <div class="container about-page">
+            <img src="img/jumbo.jpg" alt="">
+    <div class="contact">
+        <h1>Contact Us</h1>
+          <p>   (248) 671-3144 <br>
+            44529 San Fransisco, CA 94158 <br>
+            info@thisisthebestcompany.mobile 
+            </p>
```

Comment:
```suggestion
                    <h1>Contact Us</h1>
          <p>   (248) 671-3144 <br>
            44529 San Fransisco, CA 94158 <br>
            info@thisisthebestcompany.mobile 
```
Great job with completing a stretch goal and added a contact page. My only recommended change would be to wrap this section in an <address> tag. This improves the ability for a search engine to serve your website to people who are looking based on locaiton.

```suggestion
                    <h1>Contact Us</h1>
                    <address>
          <p>   (248) 671-3144 <br>
            44529 San Fransisco, CA 94158 <br>
            info@thisisthebestcompany.mobile </p>
            </address>
```
Yes, it's convention to include phone and e-mail inside the <address> tag.